With  exceptions the following i(syntactical elements) are used:
    itemization(
    iti(try): The tt(try)-block surrounds statements in which exceptions may
be generated (the parlance is for exceptions to be  thrown). Example:
        verb(
    try
    {
        // statements in which exceptions may be thrown
    }
        )
    iti(throw): followed by an expression of a certain type, throws the value
of the expression as an exception. The tt(throw) statement must be executed
somewhere within the tt(try)-block: either directly or from within a function
called directly or indirectly from the tt(try)-block. Example:
        verb(
    throw "This generates a char * exception";
        )
    iti(catch): Immediately following the tt(try)-block, the tt(catch)-block
receives the thrown exceptions. Example of a tt(catch)-block receiving
tt(char *) exceptions:
        verb(
    catch (char *message)
    {
        // statements in which the thrown char * exceptions are handled
    }
        )
    )
